src/store/useProjectStore.ts

```diff
@@ -19,14 +19,24 @@ const defaultState: Omit<
 	| "projectList"
 	| "deleteProject"
 	| "createProjectFromManifest"
+	| "setEditorWidth"
 > = {
 	projectsList: [],
 	isLoadingProjectsList: true,
+	initialEditorWidth: 50,
```

Comment:
It shouldn't be here, please move to constants. There is no reason to keep it in the store.
If you think it should be here, please explain.

Reply:
not relevant, all good